tilauskasittely/editilaus_in.inc

```diff
@@ -2118,7 +2118,7 @@ while ($tietue = fgets($fd)) {
           }
 
           // jos tullaan magentosta ja alennus on tiedossa
-          if ($edi_tyyppi == "magento" and (isset($verkkokauppa_hinnatedifailista) and $verkkokauppa_hinnatedifailista == "JOO") and $editilaus_alennus > 0) {
+          if ($edi_tyyppi == "magento" and (isset($verkkokauppa_hinnatedifailista) and $verkkokauppa_hinnatedifailista == "JOO") and $editilaus_alennus >= 0) {
```

Comment:
@mikamyllynen Onko tää muutos taaksepäin yhteensopiva? Onko mahdollista, että nykyisissä rajapinnoissa tulee jotain tyhjästä poikkeavaa joka castataan nollaksi ja siten estää alennksia tulemasta vaikka niitä pitäisi tulla?

tyhjä --> defaulttialennukset
nolla --> ei alennuksia